pinot-common/src/main/java/org/apache/pinot/common/utils/DataTable.java

```diff
@@ -103,7 +103,7 @@ enum MetadataKey {
     NUM_SEGMENTS_QUERIED("numSegmentsQueried", MetadataValueType.INT),
     NUM_SEGMENTS_PROCESSED("numSegmentsProcessed", MetadataValueType.INT),
     NUM_SEGMENTS_MATCHED("numSegmentsMatched", MetadataValueType.INT),
-    NUM_CONSUMING_SEGMENTS_PROCESSED("numConsumingSegmentsProcessed", MetadataValueType.INT),
```

Comment:
Curious if this will be backward incompatible ?

Reply:
@siddharthteotia 
In the API response we where showing numConsumingSegmentsQueried which was computed using the key NUM_CONSUMING_SEGMENTS_PROCESSED. As part of this PR we are correcting **only the variable names which was incorrectly used not the API response** so as per my understanding there should not be any compatibility issue related to the same.